build/repo.targets

```diff
@@ -246,14 +246,8 @@
   </Target>
 
   <Target Name="_RemoveTimestampFromDepsFile">
-    <GetOSPlatform>
```

Comment:
Seems like this task was also missing after the bootstrapper update but I noticed it could be simplified away so I removed it.